project_forecast_line/models/forecast_line.py

```diff
@@ -140,6 +154,16 @@ def _compute_employee_forecast_line_id(self):
             else:
                 rec.employee_resource_forecast_line_id = False
 
+    def _get_resource_forecast_line_values(self):
+        data = {}
+        for d in self.env["forecast.line"].read_group(
+            [("employee_resource_forecast_line_id", "in", self.ids)],
+            fields=["forecast_hours"],
+            groupby=["employee_resource_forecast_line_id"],
+        ):
+            data[d["employee_resource_forecast_line_id"][0]] = d["forecast_hours"]
```

Comment:
Build a dictionary with both groups

```suggestion
             line_id = d["employee_resource_forecast_line_id"][0]]
             if line_id not in data:
                 data[line_id] = {}
             data[line_id][d['type']] = d["forecast_hours"]
```